src/style.css

```diff
@@ -0,0 +1,126 @@
+@font-face {
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-family: "Avenir-Book";
+}
+
+@font-face {
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+  font-family: "Avenir-Heavy";
+}
+
+.big {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  color: #3d4e61;
+}
+
+.small {
+  margin-top: 20px;
+}
+
+.biginput {
+  color: #3d4e61;
+  font-family: "Avenir-Book", sans-serif;
```

Comment:
move it to the `html` tag selector

Reply:
what do you mean by "it" ?
Do you mean to move, color and font-family to the html tag?
or only font-family? Because color seems like not applying from html tag, it should be specifed directly in "input" selector  to make it work.